Neos.Neos/Classes/Utility/User.php

```diff
@@ -16,13 +18,20 @@ class User
     /**
      * Constructs a personal workspace name for the user with the given username.
      *
-     * @deprecated with Neos 9.0 please use {@see WorkspaceNameBuilder::fromAccountIdentifier} instead.
+     * @deprecated with Neos 9.0 please use {@see WorkspaceService::createPersonalWorkspaceForUserIfMissing()} instead.
      * @param string $username
      * @return string
      */
     public static function getPersonalWorkspaceNameForUsername($username): string
     {
-        return WorkspaceNameBuilder::fromAccountIdentifier($username)->value;
+        $name = preg_replace('/[^A-Za-z0-9\-]/', '-', 'user-' . $username);
```

Comment:
isnt this logic now completely outdated? i think we HAVE to tell users to use `createPersonalWorkspaceForUserIfMissing` (and thus `findPrimaryWorkspaceNameForUser`) this static utility is now obsolete.

Reply:
It's true.. Currently this is still used by the demo site (https://github.com/neos/Neos.Demo/blob/09864afab88b715106232b8769d6359e04cf1b3f/Classes/Form/Runtime/Action/CreateUserAction.php#L65) but we can probably replace that